docs/edge-storage-api.md

```diff
@@ -21,8 +21,8 @@ $bunnyClient = new BunnyClient(
 // Provide the "(Read-Only) Password" available at the "FTP & API Access" section of your specific storage zone.
 $edgeStorageApi = new EdgeStorageAPI(
     apiKey: '6bf3d93a-5078-4d65-a437-501c44576fe6',
-    region: Region::FS,
     client: $bunnyClient,
+    region: Region::FS,
```

Comment:
note to self: changed this order in docs only; class already had correct argument order.